examples/knowledge_tuning/instructlab/knowledge_generation_and_mixing.py

```
@@ -0,0 +1,135 @@
+# %% [markdown]
+# ### Run SDG through python command (For large scale generation)
+#
+# ```python
+# python /home/lab/sdg/scripts/generate.py --ds_path {output_dir}/seed_data.jsonl --bs 8 --num_workers 8 --save_path {output_dir}/gen.jsonl --flow ../src/instructlab/sdg/flows/generation/knowledge/synth_knowledge1.5.yaml --endpoint {teacher_endpoint_url} --checkpoint_dir {output_dir}/data_checkpoints --save_freq 2
+# ```
+
+# %% [markdown]
+# ### Save the generated data into training format
+
+# %%
+from knowledge_utils import create_knowledge_regular_ds, create_knowledge_pretraining_ds
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)